lib/core/test/unit/Cardano/Wallet/Primitive/ScriptsSpec.hs

```diff
@@ -208,7 +198,8 @@ prop_markingDiscoveredVerKeys (AccountXPubWithScripts accXPub' scripts') = do
 prop_poolExtension
     :: AccountXPubWithScriptExtension
     -> Property
-prop_poolExtension (AccountXPubWithScriptExtension accXPub' scripts') = do
+prop_poolExtension (AccountXPubWithScriptExtension accXPub' scripts') =
+    all (\s -> L.length (retrieveAllVerKeyHashes s) > 0)  scripts' ==>
```

Comment:
```suggestion
    all (not . null . retrieveAllVerKeyHashes) scripts' ==>
```
What's the `==>` for?

Reply:
I require left side to be valid, ie., script with any ver kyes, otherwise this property does not make sense